content/applications/essentials/in_app_purchase.rst

```diff
@@ -1,57 +1,186 @@
 
-.. image:: in_app_purchase/image2.png
-   :align: center
+The following |IAP| services are provided by Odoo:
 
-.. tip::
-   An IAP account can be disabled by appending `+disabled` to its token.
-   Reverting this change will re-enable the account.
+- :guilabel:`Documents Digitization`: digitizes scanned or PDF vendor bills, expenses, and resumes
+  with OCR and AI.
+- :guilabel:`Partner Autocomplete`: automatically populates contact records with corporate data.
+- :guilabel:`SMS`: sends SMS text messages to contacts directly from the database.
+- :guilabel:`Lead Generation`: generates leads based on a set of criteria, and converts web visitors
+  into quality leads and opportunities.
+- :guilabel:`Snailmail`: sends customer invoices and follow-up reports by post, worldwide.
 
-IAP Portal
-==========
+For more information on every service currently available, visit the `Odoo IAP Catalog
+<https://iap.odoo.com/iap/all-in-app-services>`_.
 
-The IAP Portal is a platform regrouping your IAP Services. It is accessible from
-:menuselection:`Settings app --> Odoo IAP --> View my Services`. From there, you can view your current
-balance, recharge your credits and set a reminder when your balance falls below a threshold.
+Use IAP services
+----------------
 
-.. image:: in_app_purchase/image3.png
-   :align: center
+|IAP| services are automatically integrated with Odoo, and do **not** require users to configure any
+settings. To use a service, simply interact with it wherever it appears in the database.
 
-Get notified when credits are low
-=================================
+.. example::
+   The following flow focuses on the *SMS* |IAP| service being used from a contact's record.
 
-To be notified when it’s time to recharge your credits, you can go to your IAP Portal through
-:menuselection:`Settings app --> Odoo IAP --> View my Services`, unfold a service and check the
-Receive threshold warning option. Then, you can provide a minimum amount of credits and email
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+   This can be done by clicking the :guilabel:`📱 (phone) SMS` icon within the database.
 
-.. image:: in_app_purchase/image4.png
-   :align: center
+   .. image:: in_app_purchase/sms-icon.png
+      :align: center
+      :alt: The SMS icon on a typical contact information form located within an Odoo database.
+
+   One way to utilize the *SMS* |IAP| service with Odoo is showcased in the following steps:
+
+   First, navigate to the :menuselection:`Contacts application`, and click on a contact with a
+   mobile phone number entered in either the :guilabel:`Phone` or :guilabel:`Mobile` field of the
+   contact form.
+
+   Next, hover the mouse over the :guilabel:`Phone` or :guilabel:`Mobile` field, and a :guilabel:`📱
+   (phone) SMS` icon appears to the right.
+
+   Following this, click the :guilabel:`📱 (phone) SMS` icon, and a :guilabel:`Send SMS Text
+   Message` pop-up window appears.
```

Comment:
I tried several diff. 15 runbots but the windows all just say "Odoo" at the top- not sure if that's runbot or not, but just want to make sure the pop-up window is named correctly. 
![image](https://github.com/odoo/documentation/assets/121518652/fd383b71-0b42-4de8-8daa-79b2086a62b6)